web-backend/metrics/Cargo.toml

```diff
@@ -15,6 +15,7 @@ serde_json = "1.0" # bson serialization
 tokio = { version = "1.1.0", features = ["full"] } # old version because of mongodb driver...
 futures = "0.3.12" # async stuff
 tracing = "0.1.22" # logging
+tracing-subscriber = "0.2"
```

Comment:
looks unused

Reply:
I used that for debugging purpose but cut out before committing. What do you suggest?

Reply:
Yeah lets either remove this since its not needed or you can throw it in a `[dev-dependencies]` section so its only used when running tests